target/linux/mvebu/files-4.14/arch/arm/boot/dts/armada-385-wd-mcex2u.dts

```diff
@@ -0,0 +1,419 @@
+/*
```

Comment:
This DTS should be submitted to the upstream kernel first, then backported to OpenWrt tree.